<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>815</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>510</y>
     <width>201</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>请连接服务器！</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="finishedTalkBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>结束对话</string>
    </property>
   </widget>
   <widget class="QPushButton" name="linkBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>连接服务器</string>
    </property>
   </widget>
   <widget class="QPushButton" name="matchBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>匹配</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="serverPortText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>181</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>181</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MS Serif</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Serif'; font-size:16pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="ATK">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>720</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>攻击</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DEF">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>720</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>反击</string>
   </property>
  </widget>
  <widget class="QPushButton" name="REC">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>720</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>休息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="send">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>620</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>发送消息</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="talkText">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>620</y>
     <width>481</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Myanmar Text</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Myanmar Text'; font-size:18pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="headLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>620</y>
     <width>191</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string>这是头像！</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="informationShow">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>941</width>
     <height>101</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
